doc/nix-local-build.rst

```diff
@@ -177,71 +178,50 @@ whatever reason (e.g., to reclaim disk space or because the global
 store is corrupted), deleting this directory is safe (``build``
 will just rebuild everything it needs on its next invocation).
 
-This split motivates some of the UI choices for Nix-style local build
+This split motivates some of the UI choices for Cabal's build
 commands. For example, flags passed to ``cabal build`` are only
 applied to *local* packages, so that adding a flag to
 ``cabal build`` doesn't necessitate a rebuild of *every* transitive
 dependency in the global package store.
 
-In cabal-install 2.0 and above, Nix-style local builds also take advantage of a
-new Cabal library feature, `per-component
-builds <https://github.com/ezyang/ghc-proposals/blob/master/proposals/0000-componentized-cabal.rst>`__,
```

Comment:
So, we lose this link in this patch... It may be good to think about preserving it in some form. I don't think there's a comparable explanation in the manual itself currently.